edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/MetricsWorker.cs

```diff
@@ -40,7 +40,12 @@ public MetricsWorker(IMetricsScraper scraper, IMetricsStorage storage, IMetricsP
             this.uploader = Preconditions.CheckNotNull(uploader, nameof(uploader));
 
             this.metricFilter = new MetricTransformer()
-                .AddAllowedTags(new KeyValuePair<string, string>(MetricsConstants.MsTelemetry, true.ToString()))
+                .AddAllowedTags((MetricsConstants.MsTelemetry, true.ToString()))
+                .AddDisallowedTags(
+                    ("quantile", "0.1"),
+                    ("quantile", "0.25"),
```

Comment:
`0.25` isn't one of the quantiles listed in `MetricsHistogram` below...do we need to disallow it?

Reply:
nope, thats leftover from a previous iteration, removing it